tests/messing_around/enumerate_periodic.in.txt

```diff
@@ -0,0 +1,253 @@
+AMS_DATA
```

Comment:
Do we want to keep this directory as part of the repo?

Reply:
Yes, for now. It's related to the "inactive site" stuff. But I renamed it. We'll probably delete it after the inactive stuff is done.